common.go

```diff
@@ -29,70 +29,69 @@ type UserStore interface {
 }
 
 // Config is a generic map of services/methods to a config value.
+// map[service]map[method]T
 type Config[T any] map[string]map[string]T
 
-// Get returns the config value for the given request.
-func (c Config[T]) Get(r *rcpRequest) (v T, ok bool) {
-	if c == nil || r.Package != "rpc" {
+// returns the config value for the given request.
+func (c Config[T]) get(r *rcpRequest) (v T, ok bool) {
+	if c == nil {
 		return v, false
 	}
-	serviceCfg, ok := c[r.Service]
-	if !ok {
-		return v, false
-	}
-	methodCfg, ok := serviceCfg[r.Method]
+
+	methodCfg, ok := c[r.serviceName][r.methodName]
```

Comment:
if the name of the service is invalid, this panics

Reply:
This doesn't panic, we have same pattern in marketplace: https://goplay.tools/snippet/MkIt6rVZGft
I also added this check with invalid service name to tests and it is passing.